eng/CredScanSuppression.json

```diff
@@ -25,6 +25,34 @@
       "hash": "5ETIzLYe9aa9Xx5512Uy6gJiCZ/CL2QzS3nDfN51nz8=",
       "_justification": "Secret used by test code"
     },
+    {
```

Comment:
Can you help me understand what type of cases we are suppressing here? We should try avoid suppressing on-off test keys. The goal should be to use placeholder values any test code and/or recording files. I believe @jeremymeng is going to help start looking into these different classes of secrets to see if we can get them normalized to a subset of placeholder values (ideally one placeholder value).

Reply:
@weshaggard while working with lang team about future plan, do we need to suppress incoming ones like this? I am afraid it will accumulate too many errors before we fixed them all.

Reply:
The first four come from our crypto wrapper libraries so replacement wouldn't apply for them
https://github.com/Azure/azure-sdk-for-js/blob/349613a6bd07e4c04f2bc4a73c9a7e5a6409acd9/sdk/core/core-crypto/test/public/sha256.spec.ts#L15

Reply:
I couldn't find the new hashes in this PR 
- "5ETIzLYe9aa9Xx5512Uy6gJiCZ/CL2QzS3nDfN51nz8="
- "UyIHw8OdxjE4j2ChZgotLVNB7fRelDvB2QGJC4Ywhgg="
- "YAxfEQdwUdxTNTXg4jsXQcFv6TBmft6LmbYBUhwoKz8="
- "/BZ/xBh93a5O2c/lIPHrrLu7nMa9LETWpduQoKWadNE="
in master. Where did them come from? would it be better to add a field for containing file path?

Reply:
@sima-zhu I would rather us refresh the baseline files for new issues until we solve the problems overall. That is better then adding these hidden hash's into the suppression file. When we are done with our work we shouldn't have a baseline any longer and I'd like to see our suppression file only contain placeholder entries and perhaps some test certificate files.